contracts/near/eth-client/src/lib.rs

```diff
@@ -421,6 +620,23 @@ impl EthClient {
         );
 
         (H256(pair.0), H256(pair.1))
+    }  
+
+    // check if the author is in the validators set.
+    fn is_in_validator_set (&self, epoch_header: &BlockHeader, add: Address) -> bool{
```

Comment:
The rust code doesn't look properly formatted. Can you use rustfmt to format it!